lib/chromic_pdf/api.ex

```diff
@@ -35,6 +35,7 @@ defmodule ChromicPDF.API do
     opts =
       opts
       |> put_source(source)
+      |> rewrite_wait_for()
```

Comment:
`rewrite_wait_for` seems like a weird naming. Sounds like an issue title rather than a function name. 🤷 
But on the other hand I can't come up with a better one. `replace_wait_for_with_evaluate` or simply `handle_wait_for`. But maybe it is also ok.